pom.xml

```diff
@@ -193,6 +193,8 @@
         -->
 
         <!-- maven plugin versions -->
+        <version.plugin.archetype-packaging>3.0.1</version.plugin.archetype-packaging>
+        <version.plugin.archetype>3.0.1</version.plugin.archetype>
```

Comment:
In case it matters they _just_ released a new version of the plugin yesterday; first in a long while

Reply:
3.1.2 ? OK will update the version.